Makefile

```diff
@@ -20,8 +20,20 @@ gen: manifests generate format vet ## Generate code containing DeepCopy, DeepCop
 
 .PHONY: manifests
 manifests: controller-gen ## Generate CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) crd paths="./..." output:crd:artifacts:config=charts/qdrant-operator-crds/crds
-	cp charts/qdrant-operator-crds/crds/qdrant.io_qdrantversions.yaml charts/qdrant-operator-crds/crds-for-cluster-api/qdrant.io_qdrantversions.yaml
+	rm charts/qdrant-operator-crds/templates/management-crds/*.yaml
+	rm charts/qdrant-operator-crds/templates/region-crds/*.yaml
+	$(CONTROLLER_GEN) crd paths="./..." output:crd:artifacts:config=charts/qdrant-operator-crds/templates
+	mv charts/qdrant-operator-crds/templates/qdrant.io_qdrantversions.yaml charts/qdrant-operator-crds/templates/management-crds/
```

Comment:
Maybe not for this PR, but we want to rename QdrantVersion to QdrantRelease

Reply:
yup, there is [ticket](https://github.com/qdrant/cloud-pm/issues/1539) for that.